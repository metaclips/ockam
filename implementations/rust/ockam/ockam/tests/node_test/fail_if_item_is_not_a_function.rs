#[ockam::test]
struct A {}

fn main() {}
