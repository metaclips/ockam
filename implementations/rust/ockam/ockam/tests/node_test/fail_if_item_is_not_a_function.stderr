error: expected `fn`
 --> tests/node_test/fail_if_item_is_not_a_function.rs:2:1
  |
2 | struct A {}
  | ^^^^^^
